apps/test-viewer/src/components/Viewer.tsx

```diff
@@ -10,6 +10,8 @@ import { history } from "../history";
 import { getUiProvidersConfig } from "../UiProvidersConfig";
 import { useAuthorizationContext } from "./Authorization";
 import { ApiKeys } from "./ApiKeys";
+import { FrontendDevTools } from "@itwin/frontend-devtools";
+import { MapLayersFormats } from "@itwin/map-layers-formats";
```

Comment:
remove this import;

i'd rather you initialize FrontendDevTools in the onIModelAppInit in here, since its not tied to any specific ui feature.

Reply:
@aruniverse @saskliutas wanted to group the initializes to happen for the map-layers features.  
So `FrontendDevTools` initialize in `onIModelAppInit`, do we all agree now ?

Reply:
this should be ok, frontend dev tools has nothing to do with the map-layers stuff, will be useful to have by default